packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -1,6 +1,16 @@
 .dataviews-wrapper {
 	width: 100%;
+	min-height: calc(100% - 60px);
+	overflow: auto;
```

Comment:
Was this one necessary?

Reply:
I think so, yes. Without it, if the list is bigger than the frame's height, it will still scroll but not get the right padding applied to this container and results in having the pagination at the bottom without it. I'm not sure if I could use a better css rule..